app/views/lobby/home.scala

```diff
@@ -110,7 +110,7 @@ object home {
               ctx.blind option h2("Timeline"),
               views.html.timeline entries userTimeline,
               // userTimeline.size >= 8 option
-              a(cls := "more", href := routes.Timeline.home)(trans.more(), " »")
+              (userTimeline.size > 0) option a(cls := "more", href := routes.Timeline.home)(trans.more(), " »")
```

Comment:
Please prefer `userTimeline.nonEmpty` here

Reply:
Fixed in latest commit.